src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,159 @@
+   *            Button with white answer icon
+   *          </Button>
+   */
+  children?: React.Node,
+  /**
+   * Specify type of the button that you want to use
+   * @example <Button type="primary">
+   *            button
+   *          </Button>
+   * @see type="primary" https://styleguide.brainly.com/latest/docs/interactive.html?type="primary"#buttons
+   * @see type="primary-inverted" https://styleguide.brainly.com/latest/docs/interactive.html?type="primary-inverted"#buttons
+   * @see type="primary-blue" https://styleguide.brainly.com/latest/docs/interactive.html?type="primary-blue"#buttons
+   * @see type="primary-mint" https://styleguide.brainly.com/latest/docs/interactive.html?type="primary-mint"#buttons
+   * @see type="secondary" https://styleguide.brainly.com/latest/docs/interactive.html?type="secondary"#buttons
+   * @see type="link-button" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button"#buttons
+   * @see type="link-button-inverted" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-inverted"#buttons
+   * @see type="link-button-peach" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-peach"#buttons
+   * @see type="link-button-mustard" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-mustard"#buttons
+   * @see type="destructive" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-inverted"#buttons
+   * @see type="warning" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-inverted"#buttons
+   * @see type="facebook" https://styleguide.brainly.com/latest/docs/interactive.html?type="link-button-inverted"#buttons
+   */
+  type: ButtonTypeType,
+  /**
+   * You can render icon inside each type of button on the left side
+   * @example <Button
+   *           icon={ICON_TYPE.FB}
+   *           buttonType={BUTTON_TYPE.FB}
+   *          >
+   *            Login with Facebook
+   *          </Button>
+   */
+  icon?: ?Node,
+  /**
+   * There are three sizes options for buttons, not need to be specify, default is medium
+   * @example <Button type="primary" size="medium">
+   *            button
+   *          </Button>
+   * @see size="small" https://styleguide.brainly.com/latest/docs/interactive.html?size="small"#buttons
+   * @see size="medium" https://styleguide.brainly.com/latest/docs/interactive.html?size="medium"#buttons
+   * @see size="large" https://styleguide.brainly.com/latest/docs/interactive.html?size="large"#buttons
+   */
+  size?: ButtonSizeType,
+  /**
+   * Specify href for button, optional string
+   * @example <Button href="https://brainly.com/" size="medium" type="primary-blue">
+   *            button
+   *          </Button>
+   */
+  href?: string,
+  /**
+   * Optional boolean for disabled button
+   * @example <Button type="primary-mint" disabled>
+   *            button
+   *          </Button>
+   */
+  disabled?: ?boolean,
```

Comment:
don't use nullable values, especially for booleans

Reply:
@PinkaminaDianePie sure! on it.

Reply:
solved